test/tests/reporter/GriddedDataReporter/tests

```diff
@@ -0,0 +1,8 @@
+[Tests]
+  [gReporterexo]
+    type = Exodiff
+    input = 'dummy.i'
+    exodiff = 'dummy_out.e'
```

Comment:
I don't think anything was written to the exodus file.  Use jsondiff.

Reply:
done